packages/devtools_app/lib/src/analytics/constants.dart

```diff
@@ -3,38 +3,27 @@
 // found in the LICENSE file.
 
 // Type of events (event_category):
-const String applicationEvent = 'application'; // visible/hidden
+
 const String screenViewEvent = 'screen'; // Active screen (tab selected).
 const String selectEvent = 'select'; // User selected something.
 
-const String errorError = 'onerror'; // Browser onError detected in DevTools
-const String exceptionEvent = 'exception'; // Any Dart exception in DevTools
+//const String exceptionEvent = 'exception'; // Any Dart exception in DevTools
```

Comment:
can you file an issue to track exceptions in analytics and link to the issue in a todo here?

Reply:
Yup, will do.